### PR TITLE
[Merged by Bors] - feat: Add `integrable_add_iff_of_nonneg`

### PR DESCRIPTION
**Pull request by LorenzoLuccioli**
Add the lemma saying that if two functions are almost everywhere nonnegative then their sum is integrable iff they are both integrable. 

In particular, add the following lemmas:
1. `integrable_left_of_integrable_add_of_nonneg`
2. `integrable_right_of_integrable_add_of_nonneg`
3. `integrable_add_iff_of_nonneg`
4. `integrable_add_iff_of_nonpos`

1. and 2. are auxiliary for 3.; 4. is the same result for nonpositive functions.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
Seems okay

maintainer merge

File: Mathlib/MeasureTheory/Function/L1Space.lean [hunk 1]
```diff
@@ -706,6 +706,40 @@ lemma integrable_add_iff_integrable_left {f g : α → β} (hf : Integrable f μ
     Integrable (g + f) μ ↔ Integrable g μ := by
   rw [add_comm, integrable_add_iff_integrable_right hf]
 
+lemma integrable_left_of_integrable_add_of_nonneg {f g : α → ℝ}
+    (h_meas : AEStronglyMeasurable f μ) (hf : 0 ≤ᵐ[μ] f) (hg : 0 ≤ᵐ[μ] g)
+    (h_int : Integrable (f + g) μ) : Integrable f μ := by
+  simp_rw [Integrable, h_meas, true_and]
```
Comment (fpvandoorn) [inline Mathlib/MeasureTheory/Function/L1Space.lean:712]: Can you prove this more directly using [MeasureTheory.Integrable.mono](https://leanprover-community.github.io/mathlib4_docs/Mathlib/MeasureTheory/Function/L1Space.html#MeasureTheory.Integrable.mono)?

Comment (LorenzoLuccioli) [reply Mathlib/MeasureTheory/Function/L1Space.lean:712]: Yes, the proof becomes much shorter, thank you!

**Comment by fpvandoorn** [comment]
Thanks!

bors d+

Please merge after the CI test on the last commit has succeeded.

**Pull request closed without merging**